web/router.ex

```diff
@@ -51,6 +51,7 @@ defmodule Healthlocker.Router do
     get "/account/nhs-help", AccountController, :nhs_help
     resources "/components", ComponentController, only: [:index]
     resources "/feedback", FeedbackController, only: [:index, :create]
+    get "/messages", MessageController, :index
```

Comment:
I really prefer to use the `resources` helper when generating routes. It protects us from typos in URLs and also forces us to stay RESTful. I think it's worthwhile, even when there's only one action.